services/movie.py

```
@@ -15,13 +18,17 @@
     if actors_ids:
         queryset = queryset.filter(actors__id__in=actors_ids)
 
+    if title:
+        queryset = queryset.filter(title__icontains=title)
+
     return queryset
 
 
 def get_movie_by_id(movie_id: int) -> Movie:
     return Movie.objects.get(id=movie_id)
```

Using `get()` to retrieve a movie by ID will raise a `DoesNotExist` exception if the movie is not found. Consider handling this exception or using `get_object_or_404()` if this function is used in a view context.